labman/gui/templates/library_pooling.html

```diff
@@ -0,0 +1,276 @@
+{% extends sitebase.html %}
+
+{% block head %}
+<script type='text/javascript'>
+
+  var poolParams = {
```

Comment:
are these defaults assured to be in sync with the python library code? is there a mechanism in place to ensure they are consistent? if not, there will be unexpected behavior in the future if and when this block (or the python code) under goes maintenance.